src/epub.test.ts

```diff
@@ -1,14 +1,19 @@
-import Epub from "./epub.js";
-import {readFileSync} from 'fs';
-
 import { describe, expect, it, beforeAll } from 'vitest';
+import { readFileSync } from 'fs';
+
+import Epub from './epub.js';
 
 describe('EPub', () => {
   let epubFile: File;
 
   beforeAll(() => {
-    const epubFileData = readFileSync('./Henry James - The Death of the Lion.epub');
-    epubFile = new File([epubFileData.buffer], 'Henry James - The Death of the Lion.epub');
+    const epubFileData = readFileSync(
+      './Henry James - The Death of the Lion.epub',
+    );
+    epubFile = new File(
+      [new Uint8Array(epubFileData)],
```

Comment:
The conversion to Uint8Array should not be necessary. Please check wether your File type is from lib.dom.d.ts

Reply:
The problem is that TS says that it needed to convert.
![image](https://github.com/user-attachments/assets/390b48c0-1ace-4c8c-8c6c-3ad3108d91b4)